composer.json

```diff
@@ -1,46 +1,46 @@
 {
-    "name": "elisa/live-shopping-integration",
-    "description": "Integration between Elisa Live Shopping and Shopware",
+    "name": "sprii/live-shopping-integration",
```

Comment:
I fear this will mess up when customer update the plugin?
Maybe they need to remove it completely before updating?
Do we know what will happend when the name is changed?

Reply:
Damn, this task got lost :-(
I should have been setting up Shopware to test this...